compiler/src/main/java/com/rejasupotaro/android/kvs/internal/SchemaWriter.java

```diff
@@ -21,129 +25,144 @@ public SchemaWriter(SchemaModel model) {
 
     public void write(Filer filer) {
         try {
-            StringBuilder fqcn = new StringBuilder();
-            fqcn.append(model.getPackageName())
-                    .append(".")
-                    .append(model.getClassName());
-
-            JavaFileObject sourceFile = filer.createSourceFile(fqcn.toString(), model.getElement());
-            JavaWriter writer = new JavaWriter(sourceFile.openWriter());
-
-            writer.emitPackage(model.getPackageName());
-            writeImports(writer);
-            writer.beginType(model.getClassName(), "class", EnumSet.of(Modifier.PUBLIC, Modifier.FINAL), model.getOriginalClassName());
+            TypeSpec.Builder classBuilder = TypeSpec.classBuilder(model.getClassName());
```

Comment:
By using addFields and addMethods, this TypeSpec will be built fluently without losing readability.